test/library/encryption/helpers.go

```diff
@@ -162,6 +162,13 @@ func WaitForNextMigratedKey(t testing.TB, kubeClient kubernetes.Interface, prevK
 	if err := wait.Poll(waitPollInterval, waitPollTimeout, func() (bool, error) {
 		currentKeyMeta, err := GetLastKeyMeta(kubeClient, namespace, labelSelector)
```

Comment:
why could extend the timeout passed to the `onErrorWithTimeout` function (inside `GetLastKeyMeta`) - it already understands 429. I think the max downtime on an SNO cluster is 60s. So we could set it to 120s

`err := onErrorWithTimeout(wait.ForeverTestTimeout, retry.DefaultBackoff, transientAPIError, func() (err error) {`